packages/catalog-model/examples-relative/acme/team-a-group.yaml

```diff
@@ -16,7 +20,7 @@ metadata:
   name: breanna.davison
 spec:
   profile:
-    displayName: Breanna Davison
+    # Intentional no displayName for testing
```

Comment:
displayName is optional, but our test data is full populate so it was unnoticed that this is optional. I remove this as a test case.